Makefile

```diff
@@ -42,10 +42,24 @@ export DEPLOYMENT_TYPE = converged
 
 # Run tests
 ENVTEST_ASSETS_DIR = $(shell pwd)/testbin
-test: generate fmt vet manifests
+setup-envtest:
 	mkdir -p $(ENVTEST_ASSETS_DIR)
 	test -f $(ENVTEST_ASSETS_DIR)/setup-envtest.sh || curl -sSLo $(ENVTEST_ASSETS_DIR)/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v0.6.3/hack/setup-envtest.sh
-	source $(ENVTEST_ASSETS_DIR)/setup-envtest.sh; fetch_envtest_tools $(ENVTEST_ASSETS_DIR); setup_envtest_env $(ENVTEST_ASSETS_DIR); go test ./... -coverprofile cover.out
+	source $(ENVTEST_ASSETS_DIR)/setup-envtest.sh; fetch_envtest_tools $(ENVTEST_ASSETS_DIR); setup_envtest_env $(ENVTEST_ASSETS_DIR);
+
+test-converged:
+	$(MAKE) -B setup-envtest
```

Comment:
Why not just depend on setup-envtest target?

Reply:
- as there'll no file changes during `setup-envtest` we can end up using same environment 
- since I doubt some resources may be left behind even after `AfterSuite` I just made `setup-envtest` to always run

Reply:
But here you are running a single test, so there will be no env test reusing